src/cmd/compress_persistence.cpp

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2021 Zilliqa
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include <iostream>
+
+#include <boost/program_options.hpp>
+#include "depends/libDatabase/LevelDB.h"
+#include "libUtils/FileSystem.h"
+
+/// Should be run with working directory where folder "persistence" consisting
+/// of the individual dbs exists.
+
+enum class STATUS {
+  ERROR_UNEXPECTED = -3,
+  ERROR_UNHANDLED_EXCEPTION = -2,
+  ERROR_IN_COMMAND_LINE = -1,
+  SUCCESS = 0,
+};
+
+using namespace std;
+namespace po = boost::program_options;
+
+int main(int argc, const char* argv[]) {
+  string dbName;
```

Comment:
No reason to be here; can just be inside the try.
Then you can consider changing:
```
int main(int argc, const char* argv[]) try {
  ...
  return static_cast<int>(STATUS::SUCCESS);
}
catch (std::exception& e) {
  ...
  return static_cast<int>(STATUS::ERROR_UNHANDLED_EXCEPTION);
}
```